src/ngraph/runtime/host_tensor_view.cpp

```diff
@@ -37,6 +38,13 @@ runtime::HostTensorView::HostTensorView(const ngraph::element::Type& element_typ
         std::make_shared<ngraph::descriptor::layout::DenseTensorViewLayout>(*m_descriptor));
 
     m_buffer_size = m_descriptor->get_tensor_view_layout()->get_size() * element_type.size();
+
+    if (mem_handle != nullptr)
+    {
+        m_aligned_buffer_pool = static_cast<char*>(mem_handle);
+        return;
```

Comment:
You could use an else statement rather than return here. You don't need to change anything, just putting up as an alternative.